src/ia-book-actions.js

```diff
@@ -631,13 +644,24 @@ export default class IABookActions extends LitElement {
       console.log('IABookActions: AutoRenewed:- ', {
         ajaxResponse: event?.detail?.data,
         loanRenewResult: this.loanRenewResult,
-        secondsLeftOnLoan: Math.round(this.lendingStatus.secondsLeftOnLoan),
+        secondsLeftOnLoan: Math.round(this.lendingStatus.secondsLeftOnLoan), // stale? how to fix
```

Comment:
this `secondsLeftOnLoan` value is previous/stale one, and not reflective of autorenewed book.

line 652 below fixes this. q: do we need to keep this console value or move the console call further down to show actual renewal time left?